WriteGCode.py

```diff
@@ -192,7 +197,10 @@ def write_gcode(config, commands) -> typing.Tuple[str, cura.LayerDataBuilder.Lay
 
 				gcode = ""
 				if is_retracted:
-					gcode += "G0 F{speed} E{e:.6f}\n".format(speed=unretraction_speed * 60, e=e)
+					gcode += "G0 F{speed} E{e:.6f}".format(speed=unretraction_speed * 60, e=e)
+					if retraction_hop_enabled:
+						gcode += " Z{z:.6f}".format(z=layer_heights[layer_nr])
+					gcode += " (UN-RETRACT)\n"
```

Comment:
Similarly, this is not standard and would give a parse error in many printers.

Reply:
```suggestion
					gcode += "G11\n" #enable zhop
```

this also works for me - ender S1 Plus